kapacitor/vars.go

```diff
@@ -153,7 +153,23 @@ func groupBy(q *chronograf.QueryConfig) string {
 func field(q *chronograf.QueryConfig) (string, error) {
 	if q != nil {
 		for _, field := range q.Fields {
```

Comment:
What does this function do? I'm confused why there's a `for` loop. It looks like you only ever hit the first element and you return after that.

Reply:
I've simplified this in adb071be379ce9252a03f0c34324d12785cf3ff6